src/enums/accounts.js

```diff
@@ -29,7 +29,8 @@ const LedgerAccountType = {
   SETTLEMENT: 2,
   HUB_RECONCILIATION: 3,
   HUB_MULTILATERAL_SETTLEMENT: 4,
-  HUB_FEE: 5
+  INTERCHANGE_FEE: 5,
```

Comment:
should HUB_FEE be removed?

Reply:
Hey @elnyry-sam-k, I am not sure, they have removed it from the central-ledger so I removed it from here